### ISSUE TITLE
Translations to Shell unrunnable after publishing

### ISSUE THREAD

**Issue by error256** [Codewars/codewars.com#690]
Translations to Shell don't have shell choice list (csh/tcsh/dash/bash) after publishind and reopening and fail to run:

> No such file or directory - fork failed
> Simplified backtrace: spawn -> run_shell -> solution -> block (3 levels) in <top (required)> -> each -> block (2 levels) in <top (required)>

For now a workaround is to change the language to another one and then back.
Kumite don't have this issue.

**Comment by lynxlynxlynx** [comment]
I can confirm this with my first kata. Had to republish as a random language and then republish as shell, since there was no option for shell in the list any more.

https://www.codewars.com/kumite/5bde2cbbd9b8407668000017

**Comment by kazk** [comment]
@error256 can you post a link to it? Also, a screenshot will help since I might see something different from my account.

**Comment by error256** [comment]
I don't have a Shell translation now. Here's a Rust one:
https://www.codewars.com/kumite/5bdf090175fad9666400011e
![Missing version selector](https://user-images.githubusercontent.com/5375100/48489455-2a466500-e834-11e8-9869-a31d73e1f643.png)
It only happens when an existing translation is reopened.

**Comment by kazk** [comment]
Thanks @error256. By "reopen" do you mean when editing existing translation?

**Comment by error256** [comment]
Yes. It happens only to _existing_, _published_ translations in _some_ languages.

**Comment by kazk** [comment]
I just checked the database and that Rust translation have language version 'rust' :man_facepalming:
I've fixed it to '1.25', can you try again?

**Comment by error256** [comment]
It's still 1.15. When unpublished, it became 1.25, but when published again, it became 1.15 (without the selector).
And here's a new Shell tranlstion:
https://www.codewars.com/kumite/5bec43f045d0ed819b00012b?sel=5bec43f045d0ed819b00012b
The same bug, but the error message is slightly different from what it was back then:
```
No such file or directory - shell
Simplified backtrace: run_shell -> block (3 levels) in <top (required)> -> times -> block (2 levels) in <top (required)>
```

**Comment by kazk** [comment]
This should be fixed. The version should now stay the same when editing published kata.
